src/views/shared/base.njk

```diff
@@ -104,5 +104,13 @@
 {% block bodyEnd %}
     {% block scripts %}{% endblock %}
     <script nonce='{{ cspNonce }}' src="{{ assetsCdnPath }}/public/javascripts/application.js"></script>
-    <script nonce='{{ cspNonce }}'>window.GOVSignIn.appInit("{{ gtmId }}", "{{ analyticsCookieDomain }}", "{{ googleTagManagerPageId }}")</script>
+    <script nonce='{{ cspNonce }}'>
+        window.DI.httpStatusCode = {{ statusCode | d(200) }};
+        window.DI.journeyState = "{{ googleTagManagerPageId }}";
+        window.DI.appInit({
+            analyticsCookieDomain: "{{ analyticsCookieDomain }}",
+            uaContainerId: "{{ uaContainerId }}",
+            ga4ContainerId: "{{ ga4ContainerId }}",
```

Comment:
Do we need to also pass the `isGa4Enabled` parameter here? [see the DCMAW documentation](https://github.com/alphagov/di-ipv-dca-front/blob/main/src/assets/javascript/docs/analytics.md#:~:text=We%20then%20call,the%20current%20environment.)

Reply:
I had removed it to keep things simple and only use the new UA + GA4 configuration (letting us delete all the old code) - unless you think it's important to have it?

Reply:
as long as we're sending UA and GA4 events I can’t see a problem with removing it I think